{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "45be0d3f_01deec74",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 1003935
      },
      "writtenOn": "2021-02-08T08:02:13Z",
      "side": 1,
      "message": "The failing tests in TraceIT need to be adapted now.",
      "revId": "4de2509087c736244c1d65b4ac863b3c418870df",
      "serverId": "173816e5-2b9a-37c3-8a2e-48639d4f1153"
    }
  ]
}